### PR TITLE
Local dotnet new testing instructions

### PR DESCRIPTION
**Pull request by patridge**
There are probably ways to test this as a local NuGet source, but here are the steps I use to test the templates locally.

(Also clarified some NuGet spec fields based on their docs.)

### CODE REVIEW

File: Meadow_DotNet_SDK/readme.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Additionally, at the top of the solution is a `Directory.Build.props` file which
 </Project>
 ```
 
-One important thing to note, there is a bug in Visual Studio (Windows) that requires the `TargetFramework` to be set on all `.props` files, even though it's set here.
```
Comment (CartBlanche) [inline Meadow_DotNet_SDK/readme.md:36]: NIT: Any chance we can fix these white spaces, as it shows more changes than necessary. There seem to be 3 of them.

**Review by CartBlanche — CHANGES_REQUESTED** [review]

File: Meadow_DotNet_SDK/readme.md (hunk 1 above)
Comment (patridge) [reply Meadow_DotNet_SDK/readme.md:36]: Technically removing them was a fix. It's extra trailing whitespace that VS Code removed automatically for me. I can put them back, I guess.

Comment (patridge) [reply Meadow_DotNet_SDK/readme.md]: Done

Comment (CartBlanche) [reply Meadow_DotNet_SDK/readme.md]: Yes, I see, but it just seemed to detract from the real change. But like I said it was a nit, so we could have just left it.

**Pull request merged**